TLM/TLM/ThreadingExtension.cs

```diff
@@ -21,6 +21,8 @@
 using TrafficManager.RedirectionFramework;
```

Comment:
Remove duplicate using 😃

Reply:
This one haven't been visited yet, you see all imports and tabs are still there. I finished at `TLM/State/Options.cs`

Reply:
oh... ok :)